src/miniscript/types/extra_props.rs

```diff
@@ -269,6 +293,8 @@ impl Property for ExtData {
             max_sat_size: Some((33, 33)),
             max_dissat_size: Some((33, 33)),
             timelock_info: TimeLockInfo::default(),
+            exec_stack_elem_count_sat: Some(2), // either size <32> or <hash256> <32 byte>
```

Comment:
nit: some of these comments are slightly wrong (wrong hash, 20 bytes vs 32 bytes)